src/Sentry.Protocol/CollectionsExtensions.cs

```diff
@@ -22,12 +22,8 @@ public static ConcurrentQueue<T> EnqueueAll<T>(this ConcurrentQueue<T> target, I
         }
 
         public static void TryCopyTo<TKey, TValue>(this IDictionary<TKey, TValue> from, IDictionary<TKey, TValue> to)
+            where TKey : notnull
         {
-            if (from == null || to == null)
-            {
-                return;
-            }
-
             foreach (var kv in from)
```

Comment:
This was not covered by tests so I removed it since it doesn't appear like the dictionaries are ever set to null. Is it okay @bruno-garcia?